bookshelf/php.ini

```diff
@@ -0,0 +1 @@
+suhosin.executor.func.blacklist="escapeshellarg, escapeshellcmd, exec, highlight_file, lchgrp, lchown, link, symlink, passthru, pclose, popen, proc_close, prog_get_status, proc_nice, proc_open, proc_terminate, shell_exec, show_source, system, gc_collect_cycles, gc_enable, gc_disable, gc_enabled, getmypid, getmyuid, getmygid, getrusage, getmyinode, get_current_user, libxml_disable_entity_loader, parse_str, phpinfo, phpversion"
```

Comment:
That PR submitted. Let's not forget to stop using dev-master:
https://github.com/GoogleCloudPlatform/getting-started-php/issues/13